### DISCUSSION TITLE
Does SIGMAC enable converting to SIGMA from supported backends?

### DISCUSSION THREAD

**Discussion by wbnod** [SigmaHQ/sigma#2644] [Q&A] [answered]
Does SIGMAC allow converting a query (e.g. Splunk, ECS) into SIGMA format vs. the other way around especially in bulk? 

Tried searching, but the only references I can find are the Uncoder.IO tool. Taking it one step further, is there general availability for using SIGMA as an intermediate format to convert between SIEM formats?

**Comment by frack113** [comment]
Hi, 
Sigmac is to convet Sigma to siem query  only .
sigma -> backend -> query
I don't know if anyone has a query -> sigma good converter.
When I receive a splunk, elk or other format query I convert it by hand to sigma to keep it without being blocked by the SIEM.
The biggest problem is when people use custom field names you have to find the original...

**Comment by Neo23x0** [comment, marked as answer]
It's possible to make a cake from flour, butter and eggs. 
It's much more difficult to make flour, butter and eggs from a cake. 

https://github.com/SigmaHQ/sigma#coverage-illustration

![image](https://user-images.githubusercontent.com/2851492/152610245-ed42050f-3e7b-4d3c-bede-c38af610f4ec.png)

**Reply by wbnod** [reply]
Completely understand. Thanks. :)
